src/main/java/org/apache/sling/event/impl/jobs/JobBuilderImpl.java

```diff
@@ -64,7 +64,7 @@ public ScheduleBuilder schedule() {
         return new JobScheduleBuilderImpl(
                 this.topic,
                 this.properties,
-                UUID.randomUUID().toString(),
```

Comment:
leaves an unused import behind, but why not dropping the name from constructor?

Reply:
Right now there is no way for an API user to specify the scheduleName, but I think that there is some value in having it.
I am thinking about an API extension which exposes it.